alws/utils/copr.py

```diff
@@ -22,10 +22,14 @@ def generate_repo_config(
 ) -> str:
     # we should clean "http" protocol from host url
     clean_host_name = re.sub(r'^(http|https)://', '', settings.pulp_host)
+    repo_url = re.sub(
+        '(x86_64|aarch64|ppc64le|i386|i686|s390x)', '$basearch', repo.url
+    )
+    repo_url = re.sub('almalinux-(8|9|10)', 'almalinux-$releasever', repo_url)
```

Comment:
same as above
```
In [7]: re.sub('almalinux-(8|9|10)', 'almalinux-$releasever', url)
Out[7]: 'https://build.almalinux.org/pulp/content/builds/almalinux-$releasever-beta-s390x-AlmaLinux-8-s390x-dr/'
```
also I've noticed that we have different suffixes in urls for some products on the production
for example:
https://build.almalinux.org/product/12 - `<product_name>-AlmaLinux-8-...`
https://build.almalinux.org/product/47 - `<product_name>-almalinux-8-...`

Reply:
the almalinux-8-beta-s390x was created before COPR was implemented fully, so repos there have different name format